examples/anyFS/test/stage1/firmware.json

```diff
@@ -0,0 +1,37 @@
+[{
+    "type": "esp32-fota-http",
+    "version": 1,
+    "host": "github.com",
+    "port": 443,
+    "bin": "/tobozo/esp32FOTA/blob/tests/examples/anyFS/test/stage1/1.nosecurity.ino.bin?raw=true"
```

Comment:
Could we reference the main repo please

Reply:
- Solution 1: Deleting the `examples/anyFS/test/stage1` folder and updating the URLs in the ino files to `http://server/fota/fota.json` lorem ipsum values
- Solution 2: Moving the `examples/anyFS/test/stage1` folder outside the main tree (e.g. to `wiki` or `gh-pages` branch) and have the firmware.json and binaries generated by a workflow.

Reply:
I went with solution 1), the solution 2) can be implemented later

Reply:
Another option is setting up a custom domain for it .

I have a local system that lets me manage updates via a WebServer and DB , maybe I expand that to allow for a simple login and let people use it as  download source , and example bins could be stored there  ?

Reply:
Locally hosted system sounds like too much honey for the simple needs of a test suite. 

With the scope of this library covering C3, S2 and S3 models of ESP32, the different partition schemes, edge cases and experiments, the amount of logical situations to handle would explode to an unmanageable level.

The test suite is composed of 8 binary files and 1 json manifest, so exhaustively handling the whole test suite for *just one partition scheme* would require to create/compile 32 files.

I'm convinced something can be done with `workflow_dispatch` to mitigate this number.

- Use mandatory workflow dispatch form entries `esp32_model` `test_suite_url` and `test_suite_base path`
- Use `$test_suite_url` and `$test_suite_base path` to generate a custom firmware.json
- For each test-suite sketch (github matrix run):
  - Use `$test_suite_url` and `$test_suite_base path` to generate config.h
  - Compile sketch using `$esp32_model` as architecture (and optionnally `$partition_scheme`?)
  - Save the compiled binaries (may also have a spiffs binary) as artefacts
- Gather all artefacts from the actions tab
- Deploy manually on personal/local webserver

The last step of the workflow can eventually be replaced by a more specific action, but deploying is at the limit between test and production contexts and it feels like this workflow shouldn't oversetp.